ansible/roles/stack-sunbird/templates/uci_api.env

```diff
@@ -31,3 +31,5 @@ UCI_CORE_BASE_URL=http://inbound-service.{{namespace}}.svc.cluster.local:8085
 #FusionAuth
 FUSIONAUTH_KEY={{fusionauth_service_admin_key}}
```

Comment:
Make sure that you've added enough comment for the variable, if the name is not self explanatory. For instance, here, what's the admin key? Is there any simple way to create the key ? `openssl rand -hex 20` ? what should be the format?
Refer: https://github.com/project-sunbird/sunbird-devops/blob/0218e3a7708eb797f8537ba143f4cc1ddd295156/private_repo/ansible/inventory/dev/Core/secrets.yml#L69